### DISCUSSION TITLE
warning message - multicell_withSecondaryGnb

### DISCUSSION THREAD

**Discussion by benemau** [omnetpp/omnetpp#1085] [Q&A]
Good morning,

I'm stuck with the omnetpp.ini in the folder dual connectivity-multicell. I modified the sampling time of the doubleconnection-CBR-UL to 0.000125s and the packet size to 3000. When running the omnetpp.ini, after few seconds I receive this message:

Warning: Context component (NRRxPdcpEntity)NRRxPdcpEntity Cid: 67174402 is deleting an object named "CBR-frag-0" it doesn't own, owner is (NRPdcpRrcEnb)pdcpRrc; set allow-object-stealing-on-deletion=true to disable this error message -- in module (NRRxPdcpEntity) MultiCell_withSecondaryGnb.masterGnb1.cellularNic.pdcpRrc.NRRxPdcpEntity Cid: 67174402 (id=520), at t=0.619s, event #311179

How can I solve this? 
Thank you, 
Benedetta Maurizio

**Comment by rhornig** [comment]
Quoted directly from the error message

> set allow-object-stealing-on-deletion=true to disable this error message

The model you are using was probably written for and older version of OMNeT++ and current versions are more picky regarding ownership management. But as the message suggest, you can fall back to the old les restrictive approach.

**Reply by benemau** [reply]
Hi,

I tried to modify several times the NRRxPdcpEntity.cc file, adding a line "set allow-object-stealing-on-deletion=true", but it doesn't work. 
Where can I add this line/option?

**Reply by rhornig** [reply]
in the omnetpp.ini file or on the command line. This is a so called configuration option for omnet.
